### PR TITLE
were created search bars

### PR DESCRIPTION
**Pull request by KrisMur**
- [DEMO LINK](https://KrisMur.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://KrisMur.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div>
```
Comment (mykola-koval) [inline src/index.html:14]: do you need this div?

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,34 @@
...
+      data-qa="big"
+      class="formBig">
+        <label>
+          <img src="images/Search.svg"
+          alt="search field"
+          class="image">
+          <input type="search" placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          class="searchBig">
+        </label>
```
Comment (mykola-koval) [inline src/index.html:25]:
```suggestion
        <label>
          <img 
            src="images/Search.svg"
            alt="search field"
            class="image"
          >
            
          <input 
            type="search" 
            placeholder="Try &quot;Los Angeles&quot;"
            data-qa="keypress"
            class="searchBig"
          >
        </label>
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  font-weight: 300;
+  padding-left: 33px;
+  line-height: 19px;
+  font-size: 14px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border-radius: 4px;
+}
```
Comment (mykola-koval) [inline src/style.css:88]: image
read through the checklist once more and check if you have any ways to improve the code 
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md

**Review by mykola-koval — CHANGES_REQUESTED** [review]

**Review by Olexandr6 — APPROVED** [review]
Good job🔥

**Pull request closed without merging**